cfg/config.go

```diff
@@ -4,27 +4,111 @@ import (
 	"encoding/json"
 	"io/ioutil"
 	"os"
+	"path/filepath"
 
 	"github.com/muesli/beehive/bees"
+	gap "github.com/muesli/go-app-paths"
+	log "github.com/sirupsen/logrus"
 )
 
+const appName = "beehive"
+
+var cfgFileName = "beehive.conf"
+var lookupFunc = Lookup
```

Comment:
On the one hand this is great because it allows for better tests, but on the other hand I'm a bit wary because it adds another layer of abstraction to the code here.

Reply:
Makes sense. An integration test that doesn't require source code changes would probably be a better places for something like this.